### PR TITLE
ci: codecov and badges

### PR DESCRIPTION
**Pull request by bacv**
Using grcov and codecov.io to generate test coverage reports and related badges.
In addition this PR adds crates.io, docs.rs and CI badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+[crates-badge]: https://img.shields.io/crates/v/waku-bindings.svg
+[crates-url]: https://crates.io/crates/waku-bindings
+[docs-badge]: https://docs.rs/waku-bindings/badge.svg
+[docs-url]: https://docs.rs/waku-bindings
+[actions-badge]: https://github.com/waku-org/waku-rust-bindings/workflows/CI/badge.svg
+[actions-url]: https://github.com/waku-org/waku-rust-bindings/actions/workflows/main.yml?query=workflow%3ACI+branch%3Amaster
+[codecov-badge]: https://codecov.io/github/waku-org/waku-rust-bindings/branch/main/graph/badge.svg?token=H4CQWRUCUS
+[codecov-url]: https://codecov.io/github/waku-org/waku-rust-bindings
+
```
Comment (danielSanchezQ) [inline README.md:16]: 💘

**Comment by danielSanchezQ** [comment]
@bacv apparently the main test is not being run:
https://app.codecov.io/gh/waku-org/waku-rust-bindings/tree/ci%2Fcodecov/waku-bindings/tests
Could we add that test for the coverage? Not sure if it is gonna work though.

**Comment by bacv** [comment]
> @bacv apparently the main test is not being run: https://app.codecov.io/gh/waku-org/waku-rust-bindings/tree/ci%2Fcodecov/waku-bindings/tests Could we add that test for the coverage? Not sure if it is gonna work though.

Fixed it. The test was marked with `#[ignore]`, so for now codecov tests are being run with `cargo test --all-features -- --ignored`. That's something that might need a change in a future.
Anyway, after the update you can see a bunch of `Message content: Hi from 🦀!` lines in a build log, and I consider this a huge success.

**Review by danielSanchezQ — APPROVED** [review]
Awesome, thanks!!

**Pull request merged**